### ISSUE TITLE
error: module-Elmer-source-git-no-commit-or-tag

### ISSUE THREAD

**Issue by kevinsmia1939** [flathub/flatpak-builder-lint#85]
I got this error.
```
{
    "errors": [
        "module-Elmer-source-git-no-commit-or-tag"
    ],
    "warnings": [
        "module-SCOTCH-buildsystem-is-plain-cmake",
        "module-TBB-buildsystem-is-plain-cmake",
        "module-VTK-9-buildsystem-is-plain-cmake",
        "module-Elmer-buildsystem-is-plain-cmake",
        "module-Trilinos-buildsystem-is-plain-cmake"
    ]
}
If you think problems listed above are a false positive, please report it here:
  https://github.com/flathub/flatpak-builder-lint
```
I have the tag and commit in it. But still error in Elmer module.
```
    sources:
      - type: git
        url: https://github.com/ElmerCSC/elmerfem.git
        tag: release-9.0
        commit: 9352634829dfde24439000571d64438d8c683264
```

https://github.com/flathub/flathub/pull/2992

**Comment by hfiguiere** [comment]
See my comment on the PR.

**Comment by kevinsmia1939** [comment]
Solved, no commit or tag in other modules.
